terraform/state.go

```diff
@@ -1106,7 +1106,32 @@ func (m *ModuleState) Orphans(c *config.Config) []string {
 	}
 
 	result := make([]string, 0, len(keys))
-	for k, _ := range keys {
+	for k := range keys {
+		result = append(result, k)
+	}
+
+	return result
+}
+
+// OrphanOutputs returns a list of outputs that are in the State but aren't
+// present in the configuration itself.
+func (m *ModuleState) OrphanOutputs(c *config.Config) []string {
```

Comment:
Could we call this `RemovedOutputs`, as a step towards that terminology change?

Reply:
sure!